bgpd/bgp_vty.c

```diff
@@ -14604,28 +14604,6 @@ static int lcommunity_list_unset_vty(struct vty *vty, int argc,
 #if CONFDATE > 20191005
 CPP_NOTICE("bgpd: remove deprecated 'ip large-community-list <(1-99)|(100-500)|standard|expanded> <deny|permit> <LINE|AA:BB:CC>' command")
 #endif
-DEFUN (lcommunity_list_standard,
-       bgp_lcommunity_list_standard_cmd,
```

Comment:
If removing then I suggest to rename `bgp_lcommunity_list_name_standard1_cmd` to `bgp_lcommunity_list_name_standard_cmd` to clean the code a little bit.

Reply:
Thanks for pointing this out Donatas (@ton31337). I've made the recommended changes.